src/test/java/org/jenkinsci/plugins/argusnotifier/BuildMetricFactoryTest.groovy

```diff
@@ -43,4 +48,36 @@ class BuildMetricFactoryTest extends Specification {
         "myfolder"    | "myfolder." + FULL_JOB_NAME + ": " + BuildMetricFactory.NUMERIC_BUILD_STATUS_LABEL
         ""            | FULL_JOB_NAME + ": " + BuildMetricFactory.NUMERIC_BUILD_STATUS_LABEL
     }
+
+    def 'Total build time metric has git commit tag'() {
+        given:
+        long metricTimestamp = 42L
+        long totalDuration = 5L
+        int buildNumber = 349506
+        String scope = "testScope"
+        String jobFolderName = "a_jobFolderName"
+        folder.fullName >> jobFolderName
+        String commitTag = "GIT_COMMIT"
+        ImmutableMap.Builder<String, String> mapBuilder = ImmutableMap.<String, String>builder().put(commitTag,commitId);
+        Map<String, String> envVars = mapBuilder.build();
```

Comment:
No need for the builder here and you can use constants instead of needing `commitTag` on line 60 (this change removes the need for the other 2 lines above).
```suggestion
        Map<String, String> envVars = [(TagFactory.Tag.GIT_COMMIT.toString()): commitId]
```